ps-pxc-dist/Jenkinsfile.groovy

```diff
@@ -0,0 +1,233 @@
+            name: 'REMOVE_LOCKFILE'
+        )
+        booleanParam(
+            defaultValue: false,
+            description: "Check to remove sources and binary version if equals pushing",
+            name: 'REMOVE_BEFORE_PUSH'
+        )
+        booleanParam(
+            defaultValue: false,
+            description: "Skips push-to-rpm-repository stage",
+            name: 'SKIP_RPM'
+        )
+        booleanParam(
+            defaultValue: false,
+            description: "Skips push-to-apt-repository stage",
+            name: 'SKIP_APT'
+        )
+        booleanParam(
+            defaultValue: false,
+            description: "Skips sync-repos-to-production stage",
+            name: 'SKIP_SYNC'
+        )
+        booleanParam(
+            defaultValue: false,
+            description: "Skips clear-cdn-cache stage",
+            name: 'SKIP_CDN'
+        )
+        booleanParam(
+            defaultValue: false,
+            description: "Skips sync-production-downloads stage",
+            name: 'SKIP_PRODUCTION_DOWNLOADS'
+        )
+        booleanParam(
+            defaultValue: false,
+            description: "Skips refresh-downloads-area stage",
+            name: 'SKIP_PRODUCTION_REFRESH'
+        )
+    }
+    options {
+        skipDefaultCheckout()
+        disableConcurrentBuilds()
+    }
+
+    stages {
+        stage('push-to-rpm-repository') {
+            when {
+                expression { params.SKIP_RPM == false }
+            }
+            steps {
+                sh '''
+                    wget https://raw.githubusercontent.com/Sudokamikaze/jenkins-pipelines/DISTMYSQL-36/ps-pxc-dist/rpm_release.sh
+                    bash -xe rpm_release.sh
+                '''
+            }
+        }
+        stage('push-to-deb-repository') {
+            when {
+                expression { params.SKIP_APT == false }
+            }
+            steps {
+                sh '''
+                    wget https://raw.githubusercontent.com/Sudokamikaze/jenkins-pipelines/DISTMYSQL-36/ps-pxc-dist/apt_release.sh
+                    bash -xe apt_release.sh
+                '''
+            }
+        }
+        stage('sync-repos-to-production') {
+            when {
+                expression { params.SYNC == false }
+            }
+            steps {
+                sh '''
+                    cd /srv/repo-copy
+                    REPO=$(echo ${REPOSITORY} | tr '[:upper:]' '[:lower:]' )
+                    RSYNC_TRANSFER_OPTS="-avt  --delete --delete-excluded --delete-after --progress"
+                    rsync ${RSYNC_TRANSFER_OPTS} --exclude=*.sh --exclude=*.bak /srv/repo-copy/${REPO}/* 10.10.9.209:/www/repo.percona.com/htdocs/${REPO}/
+                '''
+            }
+        }
+        stage('clear-cdn-cache') {
+            when {
+                expression { params.SKIP_CDN == false }
+            }
+            steps {
+                sh '''
```

Comment:
https://github.com/Percona-Lab/jenkins-pipelines/blob/master/vars/sync2Prod.groovy#L66 please look howit is done for PMM team